File: haru-engine/src/main/java/haru/engine/HaruEngine.java
package haru.engine;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

/**
 * @author takeshi
 */
@SpringBootApplication
@EnableDiscoveryClient
public class HaruEngine {

  public static void main(String[] args) {
    SpringApplication.run(HaruEngine.class, args);
  }

}

File: haru-engine/src/main/kotlin/haru/engine/HaruEngine.kt
package haru.engine

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

/**
 * @author takeshi
 */
@SpringBootApplication
@EnableEurekaClient
open class HaruEngine {

  companion object {
    @JvmStatic
    fun main(args: Array<String>) {
      SpringApplication.run(HaruEngine::class.java, *args)
    }
  }

}

